src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -190,9 +189,10 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
             .order_by("-date")
             .distinct()
         )
-        queryset.prefetch_related(
-            "projects__release_thresholds"
-        )  # maybe prefetch "deploy_set" as well?
+        # prefetching the release_thresholds via the projects model
+        queryset.prefetch_related("projects__release_thresholds__environment")
+        queryset.prefetch_related("releaseprojectenvironment_set")
+        queryset.prefetch_related("deploy_set")
```

Comment:
Where is all this prefetched data used? It's fine to do it if necessary, I just didn't see it being used explicitly.

Reply:
Pushed up change so we're now filtering through our querysets via python and NOT via another query